core/src/main/java/tc/oc/pgm/command/ScoreLimitCommand.java

```diff
@@ -0,0 +1,38 @@
+package tc.oc.pgm.command;
+
+import static net.kyori.adventure.text.Component.translatable;
+
+import app.ashcon.intake.Command;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.format.NamedTextColor;
+import tc.oc.pgm.api.Permissions;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.api.module.exception.ModuleLoadException;
+import tc.oc.pgm.score.ScoreMatchModule;
+import tc.oc.pgm.util.Audience;
+
+public final class ScoreLimitCommand {
+
+  @Command(
+      aliases = {"scorelimit", "sl"},
+      desc = "Set the score limit",
+      usage = "[number]",
+      help = "Amount must be an integer",
+      flags = "r",
```

Comment:
What is the flag r doing?

Reply:
Actually idk, I just copied from the TimeLimitCommand class since it is similar

Reply:
The flag r controls the boolean `reset` you have in your method signature